zerver/models.py

```diff
@@ -1523,6 +1523,11 @@ class UserBaseSettings(models.Model):
     COLOR_SCHEME_CHOICES = [COLOR_SCHEME_AUTOMATIC, COLOR_SCHEME_NIGHT, COLOR_SCHEME_LIGHT]
     color_scheme = models.PositiveSmallIntegerField(default=COLOR_SCHEME_AUTOMATIC)
 
+    BUTTON_LABEL_TEXT = 1
+    BUTTON_LABEL_ICON = 2
+    BUTTON_LABEL_CHOICES = [BUTTON_LABEL_TEXT, BUTTON_LABEL_ICON]
+    button_label = models.PositiveSmallIntegerField(default=BUTTON_LABEL_TEXT)
```

Comment:
`button_label` doesn't sound like a good name for this setting. Perhaps `top_left_sidebar_buttons_style`?